### PR TITLE
Fix `daml3-script` `query` question timeouts

### PR DESCRIPTION
**Pull request by akrmn**

### CODE REVIEW

File: daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala [hunk 1]
```diff
@@ -1205,7 +1205,7 @@ object ScriptF {
       case ("SubmitTree", 1) => parseSubmit(v, stackTrace).map(SubmitTree(_))
       case ("TrySubmit", 1) => parseSubmit(v, stackTrace).map(TrySubmit(_))
       case ("TrySubmitConcurrently", 1) => parseSubmitConcurrently(v, stackTrace)
-      case ("Query", 1) => parseQuery(v)
+      case ("QueryACS", 1) => parseQueryACS(v)
```
Comment (akrmn) [inline daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala:1208]: the bug fix was `s/"Query"/"QueryACS"/`, all the other renamings in this file are only for consistency

Comment (samuel-williams-da) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala:1208]: How on earth did the integration tests not catch this...

File: compiler/damlc/tests/BUILD.bazel [hunk 2]
```diff
@@ -1241,3 +1241,13 @@ daml_compile(
...
 )
+
+daml_compile(
+    name = "query-test",
+    srcs = ["daml-test-files/Query.daml"],
+    dependencies = [
+        "//daml-script/daml3:daml3-script-2.dev.dar",
+    ],
+    target = "2.dev",
+    visibility = ["//visibility:public"],
```
Comment (samuel-williams-da) [inline compiler/damlc/tests/BUILD.bazel:1252]: You can probably just make this an integration test and use the `-- @ SCRIPT-V2 flag`

**Review by samuel-williams-da — APPROVED** [review]
Thanks for fixing this... I feel rather silly for missing it

File: daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala (hunk 1 above)
Comment (paulbrauner-da) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala:1208]: It was caught by repl-functests, some by _some_ integration test at least.

Comment (akrmn) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala:1208]: somehow none of our `daml-test-files` called `query`!

File: compiler/damlc/tests/BUILD.bazel (hunk 2 above)
Comment (akrmn) [reply compiler/damlc/tests/BUILD.bazel:1252]: it's both, I think it's worth it to test this on the gRPC ledger client too

**Pull request merged**